docs/site/Concepts.md

```diff
@@ -13,24 +14,24 @@ responsibilities for typical API and/or Microservice applications.
 
 - [**Application**](Application.md): The central class for setting up all of
   your module’s components, controllers, servers and bindings. The Application
-  class extends [Context](Context.md) and provides the controls for starting and
-  stopping its associated servers.
+  class extends [Context](./Context.md) and provides the controls for starting
```

Comment:
nitpick, it's ok to keep as `Context.md` for consistency

Reply:
Actually I had updated to relative paths, but undid it after seeing a lot more of non-breaking non-relative paths. It would be inconsistent to update only some of them in this PR; and I did not want to get into updating everything, you never know what new problems might be encountered. I wanted to make this PR a quick one. Let's keep "use of relative paths on all doc pages" as a task for another PR.

Reply:
Oh, I got what you mean. Updating.